src/aws/messages.rs

```diff
@@ -24,7 +33,7 @@ impl RekognitionResponse {
         let labels: HashSet<String> = self
             .ModerationLabels
             .iter()
-            .map(|l| l.ParentName.clone())
+            .map(|l| l.top_category().to_owned())
             .filter(|l| !l.is_empty())
```

Comment:
Is this line necessary since a empty check is occurring in the `top_category` fn?